python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -349,15 +344,31 @@ def reset(self, errors_fatal=False):
             if not self.provider:
                 self.provider = _get_node_provider(self.config["provider"],
                                                    self.config["cluster_name"])
-            # Check whether we can enable the resource demand scheduler.
-            if "available_node_types" in self.config:
-                self.available_node_types = self.config["available_node_types"]
-                self.resource_demand_scheduler = ResourceDemandScheduler(
+
+            self.available_node_types = self.config["available_node_types"]
+            upscaling_speed = self.config.get("upscaling_speed")
+            aggressive = self.config.get("autoscaling_mode") == "aggressive"
+            target_utilization_fraction = self.config.get(
+                "target_utilization_fraction")
+            if upscaling_speed:
+                upscaling_speed = float(upscaling_speed)
```

Comment:
Can we add log WARN messages about how these options are translated into the new upscaling speed?